api-reference/endpoint/query.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: "Check Software EOL"
+api: "POST https://eol.xeol.com/api/v1/query"
+description: "This endpoint checks if the provided software identifier(s) are end-of-life"
+---
+
+### Body
+
+<ParamField body="identifiers" type="object[]" required="true">
+  list of software identifiers to check for end-of-life status
+
+  <Expandable title="identifier">
+    <ResponseField name="type" type="string">
+      Identifier type for this software.
+      
+      Can be either `cpe` `purl` `custom`
+    </ResponseField>
+
+    <ResponseField name="identifier" type="string">
+      Unique identifier string for the software.
+      
+      If custom, just use the software's name like this: `redis`
+    </ResponseField>
+
+    <ResponseField name="version" type="string">
+      Version for this software
+    </ResponseField>
+
+    <ResponseField name="lts" type="boolean">
+      Whether the software has `Long Term Support`
+
+      If left out then it will default to `false`
+    </ResponseField>
+
+    <ResponseField name="els" type="UUID">
+      UUID for this software's `Extended Lifecycle Support`
+
+      If left out then it will default to `Standard Support`
+    </ResponseField>
+  </Expandable>
+</ParamField>
+
+### Response
+
+<ResponseField name="results" type="object[]">
+  The contents of the user group.
+
+  <Expandable title="result">
+    <ResponseField name="identifier" type="string">
+      Software identifier string for the matched artifact
+    </ResponseField>
+
+    <ResponseField name="version" type="string">
```

Comment:
is version here the version of the `entity`?

Reply:
yes